engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/JoinGameScreen.java

```diff
@@ -352,6 +348,22 @@ public Boolean get() {
                 refresh();
             });
         }
+
+        UIButton broadCastServerButton = find("broadcast", UIButton.class);
+        if (broadCastServerButton != null) {
+            broadCastServerButton.bindText(new ReadOnlyBinding<String>() {
+                @Override
+                public String get() {
+                    if (!broadCastServer.isBroadCastTurnedOn()) {
```

Comment:
In case `broadCastServer` cannot be injected this will be `null`, leading to a NullPointerException. That probably breaks the UI screen then.